### ISSUE TITLE
Why does the dataset not have a validation set

### ISSUE THREAD

**Issue by m1777i** [chenjie04/Dynamic-YOLO#2]
Hello, I would like to ask yolo if there is no validation set, is the validation set empty during the training phase or is the validation set the same data as the test set, and if it is the same will it have an effect on the experimental results? Is the final experimental result obtained from the test set?I would be very grateful if you could answer my questions.

**Comment by chenjie04** [comment]
> Hello, I would like to ask yolo if there is no validation set, is the validation set empty during the training phase or is the validation set the same data as the test set, and if it is the same will it have an effect on the experimental results? Is the final experimental result obtained from the test set?I would be very grateful if you could answer my questions.

Yes, there is no a validation set in the official release. The validation set is the same data as the test set. The experimental results reported in the paper are validation performance.

**Comment by m1777i** [comment]
> > Hello, I would like to ask yolo if there is no validation set, is the validation set empty during the training phase or is the validation set the same data as the test set, and if it is the same will it have an effect on the experimental results? Is the final experimental result obtained from the test set?I would be very grateful if you could answer my questions.
> 
> Yes, there is no a validation set in the official release. The validation set is the same data as the test set. The experimental results reported in the paper are validation performance.

Okay, thank you very much for your reply.
